building/Windows.md

```diff
@@ -0,0 +1,133 @@
+# Windows
+
+If we want we can build Phoenix-rtos images for all targets and run `ia32-generic-qemu` on Windows with Cygwin and WSL.

```

Comment:
I would prefer to use impersonal constructions if personal ones are not necessary.
For example:
```suggestion
It is possible to build Phoenix-rtos images for all targets and run `ia32-generic-qemu` on Windows using Cygwin or WSL.

```
I'm not sure if we have a defined way of maintaining documentation, and perhaps it would be worth discussing?